language/stdlib/modules/LibraVMConfig.move

```diff
@@ -1,33 +1,30 @@
 address 0x1 {
 
+/// This module defines structs and methods to initialize VM configurations,
+/// including different costs of running the VM.
 module LibraVMConfig {
     use 0x1::LibraConfig::{Self, LibraConfig};
     use 0x1::LibraTimestamp;
     use 0x1::CoreAddresses;
     use 0x1::Roles;
 
-    // The struct to hold all config data needed to operate the LibraVM.
-    // * publishing_option: Defines Scripts/Modules that are allowed to execute in the current configuration.
```

Comment:
Removed documentation for `publishing_option` field since it has grown up and become its own module.